[2025-02-12T22:29:12.011+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-12T22:29:12.023+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: crypto_data_collection.fetch_and_store_crypto_data scheduled__2023-10-03T23:06:00+00:00 [queued]>
[2025-02-12T22:29:12.029+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: crypto_data_collection.fetch_and_store_crypto_data scheduled__2023-10-03T23:06:00+00:00 [queued]>
[2025-02-12T22:29:12.029+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-02-12T22:29:12.041+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): fetch_and_store_crypto_data> on 2023-10-03 23:06:00+00:00
[2025-02-12T22:29:12.047+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=13499) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-12T22:29:12.048+0000] {standard_task_runner.py:72} INFO - Started process 13504 to run task
[2025-02-12T22:29:12.047+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'crypto_data_collection', 'fetch_and_store_crypto_data', 'scheduled__2023-10-03T23:06:00+00:00', '--job-id', '4816', '--raw', '--subdir', 'DAGS_FOLDER/crypto_dag.py', '--cfg-path', '/tmp/tmp6da85ru9']
[2025-02-12T22:29:12.049+0000] {standard_task_runner.py:105} INFO - Job 4816: Subtask fetch_and_store_crypto_data
[2025-02-12T22:29:12.087+0000] {task_command.py:467} INFO - Running <TaskInstance: crypto_data_collection.fetch_and_store_crypto_data scheduled__2023-10-03T23:06:00+00:00 [running]> on host be7ba0dd3a47
[2025-02-12T22:29:12.184+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='crypto_data_collection' AIRFLOW_CTX_TASK_ID='fetch_and_store_crypto_data' AIRFLOW_CTX_EXECUTION_DATE='2023-10-03T23:06:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-03T23:06:00+00:00'
[2025-02-12T22:29:12.184+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-12T22:29:12.587+0000] {logging_mixin.py:190} INFO - Données récupérées : {'Response': 'Error', 'Message': 'You are over your rate limit please upgrade your account!', 'HasWarning': False, 'Type': 99, 'RateLimit': {'calls_made': {'second': 1, 'minute': 27, 'hour': 3399, 'day': 4293, 'month': 4293, 'total_calls': 4293}, 'max_calls': {'second': 20, 'minute': 300, 'hour': 3000, 'day': 7500, 'month': 11000}}, 'Data': {}}
[2025-02-12T22:29:12.590+0000] {logging_mixin.py:190} INFO - Erreur lors de l'insertion dans PostgreSQL : string indices must be integers, not 'str'
[2025-02-12T22:29:12.604+0000] {logging_mixin.py:190} INFO - Données uploadées dans MinIO.
[2025-02-12T22:29:12.604+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-12T22:29:12.613+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-12T22:29:12.614+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=crypto_data_collection, task_id=fetch_and_store_crypto_data, run_id=scheduled__2023-10-03T23:06:00+00:00, execution_date=20231003T230600, start_date=20250212T222912, end_date=20250212T222912
[2025-02-12T22:29:12.662+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-12T22:29:12.677+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-12T22:29:12.682+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
